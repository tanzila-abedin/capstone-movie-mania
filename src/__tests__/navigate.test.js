import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Navigate from '../Components/navigate';

test('Navbar component is defined', () => {
  const component = render(
    <BrowserRouter>
      <Navigate />
    </BrowserRouter>,
  );
  expect(component).toBeDefined();
});

test('Navbar snapshot testing', () => {
  const nav = renderer.create(
    <BrowserRouter>
      <Navigate />
    </BrowserRouter>,
  ).toJSON();
  expect(nav).toMatchSnapshot();
});
