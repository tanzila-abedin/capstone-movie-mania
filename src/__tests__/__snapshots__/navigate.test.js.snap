// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot testing 1`] = `
<nav
  className="navHead navbar navbar-light"
>
  <div
    className="container"
  >
    <p
      className="navbar-brand mb-0 text-warning nav-text font-weight-normal"
    >
      Movie Mania
    </p>
    <a
      className="nav-text font-weight-normal"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </div>
</nav>
`;
